protos/telemetry/telemetry.proto

```diff
@@ -603,6 +683,24 @@ message DistanceSensor {
     float current_distance_m = 3 [(mavsdk.options.default_value)="NaN"]; // Current distance reading, NaN if unknown.
 }
 
+// Raw Pressure message type.
+message RawPressure {
+    uint64 timestamp_us = 1; // Timestamp in microseconds (UNIX Epoch time or time since system boot, to be inferred)
+    int32 absolute_pressure = 2; // Absolute pressure (raw)
+    int32 differential_pressure_first = 3; // Differential pressure 1 (raw, 0 if non-existent)
+    int32 differential_pressure_second = 4; // Differential pressure 2 (raw, 0 if non-existent)
+    int32 temperature = 5; // Raw temperature measurement (raw)
+}
+
+// Scaled Pressure message type.
+message ScaledPressure {
+    uint64 timestamp_us = 1; // Timestamp (time since system boot)
+    float absolute_pressure_hpa = 2; // Absolute pressure in hPa
+    float differential_pressure_hpa = 3; // Differential pressure 1 in hPa
+    int32 temperature_deg = 4; // Absolute pressure temperature (in celcius)
+    int32 differential_pressure_temperature_deg = 5; // Differential pressure temperature (in celcius, 0 if not available)
+}
```

Comment:
I don't think this message is implemented for PX4. I would say there is no value implementing this message.

Reply:
@TSC21:  what do you think?

Reply:
@TSC21 ?

Reply:
Decided to remove the raw pressure for now.